x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/action/PutSearchApplicationAction.java

```diff
@@ -35,10 +40,13 @@ public PutSearchApplicationAction() {
         super(NAME, PutSearchApplicationAction.Response::new);
     }
 
-    public static class Request extends ActionRequest {
+    public static class Request extends ActionRequest implements ToXContentObject {
 
         private final SearchApplication searchApp;
+        public static ParseField SEARCH_APPLICATION = new ParseField("searchApp");
+
         private final boolean create;
+        public static ParseField CREATE = new ParseField("create");
```

Comment:
Nitpick: We usually declare the `ParseFields` together in one block rather than spread out between other assignments.

Reply:
Done in  af753f213ae0513cd019ad628cadbedc1ab7ce08